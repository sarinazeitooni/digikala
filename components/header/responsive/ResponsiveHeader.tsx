import React from 'react';
import style from './scss/responsive-header.module.scss';
import MenuIcon from '@material-ui/icons/Menu';
import NotListedLocationOutlinedIcon from '@material-ui/icons/NotListedLocationOutlined';
import ExitToAppOutlinedIcon from '@material-ui/icons/ExitToAppOutlined';
import ShoppingBasketOutlinedIcon from '@material-ui/icons/ShoppingBasketOutlined';
import SearchOutlinedIcon from '@material-ui/icons/SearchOutlined';
import ArrowBackIosOutlinedIcon from '@material-ui/icons/ArrowBackIosOutlined';
import RoomOutlinedIcon from '@material-ui/icons/RoomOutlined';
import messages from "../messages/messages";
const ResponsiveHeader = () => {
    return (
        <div className={style['mobile-header']}>
            <div className={style['mobile-header-row']}>
                <div className={style['menu-icon']}>
                    <MenuIcon/>
                </div>
                <div className={style['digikala-icon']}>
                    <img src='https://www.digikala.com/static/files/d6861d76.svg'/>
                </div>
                <div className={style['chat-box-icon']}>
                    <NotListedLocationOutlinedIcon/>
                </div>
            </div>
            <div className={style['mobile-header-row'] + " " + style['input-row']}>
                <div className={style['input-container']}>
                    <SearchOutlinedIcon/>
                    <input placeholder={messages.inputPlaceHolder} type='text'/>
                </div>
                <div className={style['login-container']}>
                    <span>{messages.logIn}</span>
                    <ExitToAppOutlinedIcon/>
                </div>
                <div className={style['basket-container']}>
                    <ShoppingBasketOutlinedIcon/>
                </div>
            </div>
            <div className={style['mobile-header-row '] + " " + style['location-row']}>
                <span className={style['location-text']}>
                    <span>{messages.location}</span>
                    <RoomOutlinedIcon/>
                </span>
                <span className={style['arrow-icon']}><ArrowBackIosOutlinedIcon/></span>
            </div>
        </div>
    )
};
export default ResponsiveHeader;