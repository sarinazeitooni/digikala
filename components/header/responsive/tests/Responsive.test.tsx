import React from 'react'
import {shallow} from 'enzyme';
import ResponsiveHeader from "../ResponsiveHeader";
describe('ResponsiveHeader component', () => {
    it('should render without throwing an error', ()=> {
        const wrapper = shallow(<ResponsiveHeader/>);
        expect(wrapper).toMatchSnapshot();
    })
});