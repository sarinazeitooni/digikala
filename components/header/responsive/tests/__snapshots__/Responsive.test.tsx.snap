// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponsiveHeader component should render without throwing an error 1`] = `ShallowWrapper {}`;
